bookstore/clone.html

```diff
@@ -0,0 +1,63 @@
+<!DOCTYPE html>
+<!--
+Copyright (c) Jupyter Development Team.
+Distributed under the terms of the Modified BSD License.
+-->
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <meta name="google" content="notranslate" />
+    <meta http-equiv="Content-Language" content="en" />
+
+    <title>Clone Landing page</title>
+    <script>
+      window.onload = function() {
+        var button = document.getElementById("mybutton");
+        button.onclick = function(event) {
+          var post_json = {
+            s3_bucket: "{{ s3_bucket }}",
+            s3_key: "{{ s3_key }}"
+          };
+          var clone_url = "{{ base_uri }}/api/bookstore/cloned/";
+          var r = document.cookie.match("\\b" + "_xsrf" + "=([^;]*)\\b");
```

Comment:
This is the literal code that we use from jupyter notebook:

```javascript
    var _get_cookie = function (name) {
        // from tornado docs: http://www.tornadoweb.org/en/stable/guide/security.html
        var r = document.cookie.match("\\b" + name + "=([^;]*)\\b");
        return r ? r[1] : undefined;
    }
```

https://github.com/jupyter/notebook/blob/b8b66332e2023e83d2ee04f83d8814f567e01a4e/notebook/static/base/js/utils.js#L800-L804

So this logic would need to live within `getCookieByName` and would need to live in this script tag since we're not packaging this at all. Would that make you feel more confident?

Reply:
See PR #88